webpack.config.js

```diff
@@ -56,6 +56,7 @@ if (minimize) {
 // jitsi-meet such as app.bundle.js and external_api.js.
 var config = {
     devServer: {
+        host: '0.0.0.0',
```

Comment:
I was working with webpack-dev-server on this PR without any problems from my first review . That's why I don't understand why `host` is suddenly necessary. As there's no source code comment explaining the need, I'll just remove it from this PR.